hydromt/models/model.py

```diff
@@ -71,7 +70,6 @@ class Model(object, metaclass=ABCMeta):
     _API = {
         "crs": CRS,
         "config": Dict[str, Any],
```

Comment:
This one should also have been removed no?

Reply:
yes good catch!